### DISCUSSION TITLE
Sqlite3 restore not possible

### DISCUSSION THREAD

**Discussion by redalert99** [dani-garcia/vaultwarden#4419] [Q&A]
Hello,

i try to restore my DB.sqlite3 and the wal file.
When i start the vaultwarden container the wal file is instantly overwritten and contains Zero data.

In the admin interface my user is displayed and had 1234 passwords. 

Can someone help me ?

Tank you

**Comment by BlackDex** [comment]
This is something for discussions not issues.
